parlai/agents/bert_ranker/bi_encoder_ranker.py

```diff
@@ -69,9 +69,11 @@ def set_vocab_candidates(self, shared):
         self.vocab_candidates will contain a [num_cands] list of strings
         self.vocab_candidate_vecs will contain a [num_cands, 1] LongTensor
         """
+        self.opt['encode_candidate_vecs'] = True
         if shared:
             self.vocab_candidates = shared['vocab_candidates']
             self.vocab_candidate_vecs = shared['vocab_candidate_vecs']
+            self.vocab_candidate_encs = shared['vocab_candidate_encs']
```

Comment:
dupe

Reply:
sorry i don't understand this comment?

Reply:
Oh I thought it was a duplicate of the line above but I misread. Please disregard